### ISSUE TITLE
Share function code snippet return error 

### ISSUE THREAD

**Issue by VincentMeert** [inertiajs/inertiajs.com#140]
on the page :  https://inertiajs.com/forms 

The code snippet under "Server-side validation" return an Error. 

```
public function share(Request $request)
    {
        return array_merge(parent::share($request), [
            'errors' => fn () => $request->session()->get('errors')
                ? $request->session()->get('errors')->getBag('default')->getMessages()
                : (object) []
        ]);
    }
```

The error is : 
` ParseError syntax error, unexpected '=>' (T_DOUBLE_ARROW), expecting ']'`

**Comment by VincentMeert** [comment]
this looks to works better : 
```
         public function share(Request $request)
    {
        return array_merge(parent::share($request), [
            'errors' => function () {
                return Session::get('errors')
                        ? Session::get('errors')->getBag('default')->getMessages()
                        : (object) [];
            },
        ]);
    }
```

**Comment by RobertBoes** [comment]
That's probably because that snippet is written with an [arrow function](https://www.php.net/manual/en/functions.arrow.php), which is available as of PHP 7.4. So the snippet does contain valid PHP, perhaps you're using an older version?

**Comment by reinink** [comment]
> That's probably because that snippet is written with an [arrow function](https://www.php.net/manual/en/functions.arrow.php), which is available as of PHP 7.4. So the snippet does contain valid PHP, perhaps you're using an older version?

Exactly this. If you're using an older version of PHP, just convert it (as you've done). 👍
